back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/importers/project_importer.rb

```diff
@@ -40,6 +41,12 @@ def preview(projects)
               log " - Participation Method: #{phase[:participation_method]}"
             end
 
+            # Skip everything else for information phases
+            if phase[:participation_method] == 'information'
+              num_phases_to_import += 1 unless phase_exists
```

Comment:
Just checking, shouldn't it be `if phase_exists` instead of `unless phase_exists`?